From: Debian Rust Maintainers <pkg-rust-maintainers@alioth-lists.debian.net>
Date: Mon, 6 May 2024 10:25:32 +0200
Subject: d-0011-cargo-remove-nghttp2

Description: remove dependency on libnghttp2-sys so it can be pruned.

Last-Update: 2023-05-17

Forwarded: not-needed
---
 vendor/curl-sys-0.4.72+curl-8.6.0/Cargo.toml | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

Index: rustc/vendor/curl-sys-0.4.74+curl-8.9.0/Cargo.toml
===================================================================
--- rustc.orig/vendor/curl-sys-0.4.74+curl-8.9.0/Cargo.toml
+++ rustc/vendor/curl-sys-0.4.74+curl-8.9.0/Cargo.toml
@@ -56,7 +56,7 @@ version = "0.3.3"
 [features]
 default = ["ssl"]
 force-system-lib-on-osx = []
-http2 = ["libnghttp2-sys"]
+http2 = []
 mesalink = []
 ntlm = []
 poll_7_68_0 = []
